plugins/wasi_nn/thirdparty/ggml/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ set(LLAMA_CUDA_PEER_MAX_BATCH_SIZE "128" CACHE STRING
 option(LLAMA_HIPBLAS                         "llama: use hipBLAS"                               OFF)
 option(LLAMA_CLBLAST                         "llama: use CLBlast"                               OFF)
 option(LLAMA_METAL                           "llama: use Metal"                                 ${LLAMA_METAL_DEFAULT})
-option(LLAMA_METAL_NDEBUG                    "llama: disable Metal debugging"                   OFF)
+option(LLAMA_METAL_NDEBUG                    "llama: disable Metal debugging"                   ON)
```

Comment:
Should we set this option in `plugins/wasi_nn/CMakeLists.txt` for better integration when updating ggml?

Reply:
Sounds better.